### PR TITLE
Feature/be 4590 sso update

### PR DESCRIPTION
**Pull request by SarpBakis23**

### CODE REVIEW

File: docs/account/my-organization/security/authentications/distribution-ldap-authentication.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+Once set up, LDAP Login allows you to control access to distributed links and adjust distribution authorization through the Distribution Profiles.
+This means you can tailor access permissions according to your organization's specific needs.
+
+If you ever need to remove LDAP Login integration, the document also provides clear instructions for doing so, ensuring that your system remains secure and up-to-date.
+
+To start, go to [My Organization](/account/my-organization) > Security screen and press the **Add New** button next to LDAP Login under the **Authentications** section.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/distribution-ldap-add-new.png' />
+
+4. The **Manage Testing Portal LDAP Login** window will open, click **Create New Authentication** button
```
Comment (SarpBakis23) [inline docs/account/my-organization/security/authentications/distribution-ldap-authentication.md:27]:
```suggestion
4. The **Manage Testing Portal LDAP Login** window will open, click **Create New Authentication** button.
```

File: docs/account/my-organization/security/authentications/distribution-ldap-authentication.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+4. The **Create New Authentication** window will open, presenting two options:
```
Comment (SarpBakis23) [inline docs/account/my-organization/security/authentications/distribution-ldap-authentication.md:29]:
```suggestion
The **Create New Authentication** window will open, presenting two options:
```

File: docs/account/my-organization/security/authentications/distribution-ldap-authentication.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    - **Create New Authentication**
+    - **Create From Existing Authentication**
+You can create new configuration or create from existing configuration. Click on the **Create New Authentication** section to create new configuration.
+Please refer the **Step 3: Create From Existing LDAP Configuration** section in the 3. General Configuration Steps.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/distribution-ldap-create-options.png' />
+
+- Fill in the details of your LDAP Configurations
```
Comment (SarpBakis23) [inline docs/account/my-organization/security/authentications/distribution-ldap-authentication.md:37]:
```suggestion
- Fill in the details of your LDAP Configurations.
```

File: docs/account/my-organization/security/authentications/sso-authentication.md [hunk 4]
```diff
@@ -109,7 +118,36 @@ Only one SSO provider can be configured at a time.
...
+
+  Appcircle allows you to create a new SSO configuration based on an existing one, ensuring a smooth and efficient setup experience. 
+
+:::caution
+
+**Important:** If you're using SAML for SSO, some identity providers, like Okta, restrict you to a single SAML Entity ID. In this case, the Create From Existing SSO feature cannot be used.
+:::
+ 
+1. Navigate to the **Organization > Security > Authentications** section on your dashboard.
+2. Select the **Add New** on the **Appcircle SSO Login**
```
Comment (SarpBakis23) [inline docs/account/my-organization/security/authentications/sso-authentication.md:131]:
```suggestion
2. Select the **Add New** on the **Appcircle SSO Login**.
```

File: docs/account/my-organization/security/authentications/sso-authentication.md [hunk 5]
```diff
@@ -109,7 +118,36 @@ Only one SSO provider can be configured at a time.
...
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/security-authentications.png' /> 
+
+3. Select the **Create New Authentication** and then select the **Create From Existing SSO Configuration**
```
Comment (SarpBakis23) [inline docs/account/my-organization/security/authentications/sso-authentication.md:135]:
```suggestion
3. Select the **Create New Authentication** and then select the **Create From Existing SSO Configuration**.
```

File: docs/account/my-organization/security/authentications/sso-authentication.md [hunk 6]
```diff
@@ -232,13 +270,13 @@ Instead of writing all the settings of OpenID, you can download the settings fil
 
 Now, upload the OpenID configuration JSON file to Appcircle and complete the configuration:
 
-1. Navigate to the **Set up OpenID Provider** screen in Appcircle, which you accessed during the SSO setup in the "General Configuration Steps."
-2. Upload the downloaded OpenID configuration JSON file to Appcircle.
+1. Navigate to the **Set up OpenID Connect Provider** screen in Appcircle, which you accessed during the SSO setup in the "General Configuration Steps."
```
Comment (SarpBakis23) [inline docs/account/my-organization/security/authentications/sso-authentication.md:273]:
```suggestion
1. Navigate to the **Set up OpenID Connect Provider** screen in Appcircle, which you accessed during the SSO setup in the "General Configuration Steps".
```

File: docs/account/my-organization/security/authentications/sso-authentication.md [hunk 7]
```diff
@@ -232,13 +270,13 @@ Instead of writing all the settings of OpenID, you can download the settings fil
...
+2. Choose the **Client secret sent as basic auth** as Client Authentication
```
Comment (SarpBakis23) [inline docs/account/my-organization/security/authentications/sso-authentication.md:274]:
```suggestion
2. Choose the **Client secret sent as basic auth** as Client Authentication.
```

File: docs/account/my-organization/security/authentications/sso-authentication.md [hunk 8]
```diff
@@ -536,12 +574,14 @@ Next, configure the callback URLs in Okta to ensure proper redirection to Appcir
...
```
Comment (SarpBakis23) [inline docs/account/my-organization/security/authentications/sso-authentication.md:578]:
```suggestion
2. Choose the **Client secret sent as basic auth** as Client Authentication.
```

File: docs/account/my-organization/security/authorization/sso-authorization.md [hunk 9]
```diff
@@ -36,14 +36,14 @@ Follow **3. Specific Provider Configuration** section to complete this steps.
 
 ### Accessing SSO Mapping Settings
 
-1. Navigate to the **Organization** section on your dashboard.
-2. Select the **Integrations** and click on the **Appcircle Login**.
+1. Navigate to the **Organization > Security > Authentications** section on your dashboard.
+2. Select the **Manage** on the **Appcircle SSO Login**
```
Comment (SarpBakis23) [inline docs/account/my-organization/security/authorization/sso-authorization.md:40]:
```suggestion
2. Select the **Manage** on the **Appcircle SSO Login**.
```

File: docs/account/my-organization/security/authorization/sso-authorization.md [hunk 10]
```diff
@@ -58,6 +58,8 @@ Follow **3. Specific Provider Configuration** section to complete this steps.
 
 <Screenshot url='https://cdn.appcircle.io/docs/assets/sso-role-mapping.png' /> 
 
+5. Finally enable SSO Authorization with the **Enable SSO Authorization** toggle.
```
Comment (SarpBakis23) [inline docs/account/my-organization/security/authorization/sso-authorization.md:61]:
```suggestion
5. Finally, enable SSO Authorization with the **Enable SSO Authorization** toggle.
```

File: docs/account/my-organization/security/authentications/store-ldap-authentication.md [hunk 11]
```diff
@@ -1,42 +1,48 @@
...
 
-- Click Create button to create your LDAP
+- The **Manage Testing Portal LDAP Login** window will open, click **Create New Authentication** button.
 
-<Screenshot url='https://cdn.appcircle.io/docs/assets/ldap-login2.png' />
+- The **Create New Authentication** window will open, presenting two options:
+    - **Create New Authentication**
+    - **Create From Existing Authentication**
+You can create a new configuration or create one from an existing configuration. Click on the **Create New Authentication** section to create new configuration.
+Please refer the **Step 3: Create From Existing LDAP Configuration** section in the 3. General Configuration Steps.
```
Comment (SarpBakis23) [inline docs/account/my-organization/security/authentications/store-ldap-authentication.md:33]: steps are missing

**Review by jhalitaksoy — APPROVED** [review]

**Pull request merged**